environ/environ.py

```diff
@@ -18,15 +18,9 @@
 import sys
 import urllib.parse as urlparselib
 import warnings
-from urllib.parse import (
-    parse_qs,
-    ParseResult,
-    unquote_plus,
-    urlparse,
-    urlunparse,
-)
+from urllib.parse import ParseResult, parse_qs, unquote_plus, urlparse, urlunparse
 
-from .compat import DJANGO_POSTGRES, ImproperlyConfigured, json, REDIS_DRIVER
+from .compat import DJANGO_POSTGRES, REDIS_DRIVER, ImproperlyConfigured, json
```

Comment:
Imported names are in the wrong order

```suggestion
from .compat import DJANGO_POSTGRES, ImproperlyConfigured, json, REDIS_DRIVER
```